packages/http/src/mocker/negotiator/__tests__/NegotiatorHelpers.spec.ts

```diff
@@ -143,16 +143,64 @@ describe('NegotiatorHelpers', () => {
 
           expect(() => {
             helpers.negotiateOptionsForInvalidRequest(httpOperation.responses);
-          }).toThrow('Request invalid but mock data corrupted. Neither schema nor example defined for 400 response.');
+          }).toThrow('Request invalid but mock data corrupted. Neither schema nor example defined for 422 response.');
         });
       });
     });
 
-    describe('and no 400 response exists', () => {
+    describe('and no 422 response exists', () => {
+      test('but a 400 response exists', async () => {
+        httpOperation = anHttpOperation(httpOperation)
+          .withResponses([
+            {
+              code: '400',
+              headers: [],
+              contents: [
+                {
+                  mediaType: chance.string(),
+                  examples: [
+                    { key: chance.string(), value: '', externalValue: '' },
+                    { key: chance.string(), value: '', externalValue: '' },
+                  ],
+                  encodings: [],
+                },
+              ],
+            },
+          ])
+          .instance();
+
+        const actualConfig = helpers.negotiateOptionsForInvalidRequest(httpOperation.responses);
+        expect(actualConfig).toHaveProperty('code', '400');
+      });
+
+      test('but a default response exists', async () => {
```

Comment:
The “but a foo exists” should be a describe. Describe is to setup the context then test is for the assertion to explain what you are actually asserting.

Reply:
I agree, but I kept the style that was there. I'm all for the change and refactor opportunistically when the occasion comes.

Reply:
My bad I thought you added this, merge away!